src/main/java/org/jabref/gui/util/IconValidationDecorator.java

```diff
@@ -32,18 +32,20 @@ public IconValidationDecorator(Pos position) {
     }
 
     @Override
-    protected Node createErrorNode() {
-        return IconTheme.JabRefIcons.ERROR.getGraphicNode();
-    }
-
-    @Override
-    protected Node createWarningNode() {
-        return IconTheme.JabRefIcons.WARNING.getGraphicNode();
+    protected Node getGraphicBySeverity(Severity severity) {
+        return switch (severity) {
+            case ERROR ->
+                    IconTheme.JabRefIcons.ERROR.getGraphicNode();
+            case WARNING ->
+                    IconTheme.JabRefIcons.WARNING.getGraphicNode();
+            default ->
+                    null;
+        };
     }
 
     @Override
     public Node createDecorationNode(ValidationMessage message) {
-        Node graphic = Severity.ERROR == message.getSeverity() ? createErrorNode() : createWarningNode();
+        Node graphic = getGraphicBySeverity(message.getSeverity());
```

Comment:
`org.controlsfx.validation.decoration.GraphicValidationDecoration` should be adapted, too.